website/docs/reference/api-tokens-and-client-keys.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: API tokens and client keys
+---
+
+Unleash has a number of tokens/keys that all have slightly different use cases. Here's the list of all of them:
+
+## API tokens
+
+:::tip
+This section describes what API tokens are. For information on how to create them, refer to the [how-to guide for creating API tokens](../user_guide/token.md).
+:::
+
+Use API tokens to connect to the Unleash server API. API tokens come in two distinct types:
+- [Admin tokens](#admin-tokens)
+- [Client tokens](#client-tokens)
+
+Both types have use [the same format](#format) but have different intended uses. API tokens are considered to be *secrets* and should _not_ be exposed to end users.
+
+### Admin tokens
+
+**Admin tokens** grant *full read and write access* to all resources in the Unleash server API. Admin tokens have access to all projects, all environments, and all global resources (find out more about [resources in the RBAC document](../user_guide/rbac.md#core-principles)).
+
+Use admin tokens to:
+- automate Unleash behavior such as creating feature toggles, projects, etc.
+- write custom Unleash UIs to replace the default Unleash admin UI
+
+Do **not** use admin tokens for:
+- [Client SDKs](../sdks/index.md): You will _not_ be able to read toggle data from multiple environments. Use [client tokens](#client-tokens) instead.
```

Comment:
Nice! This comes up a lot so thanks for including this!

Reply:
Yeah, I know 😅 Hoping this'll make it more obvious 🤞🏼